test/e2e/osa_test.go

```diff
@@ -83,4 +129,23 @@ var _ = Describe("Openshift on Azure e2e tests", func() {
 		})
 		Expect(err).ToNot(HaveOccurred())
 	})
+
+	It("should disallow PDB mutations", func() {
+		maxUnavailable := intstr.FromInt(1)
+		selector, err := metav1.ParseToLabelSelector("key=value")
+		Expect(err).NotTo(HaveOccurred())
+
+		pdb := &policy.PodDisruptionBudget{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "test",
+			},
+			Spec: policy.PodDisruptionBudgetSpec{
+				MaxUnavailable: &maxUnavailable,
+				Selector:       selector,
+			},
+		}
+
+		_, err = kubeClient(c.cs.Config.AdminKubeconfig).Policy().PodDisruptionBudgets("e2e-tests").Create(pdb)
```

Comment:
Is this `system:admin` for now? Not yet delegated?

Reply:
Yes, there is a TODO in kubeClient to fix that